prov/verbs/src/verbs_ep.c

```diff
@@ -567,6 +736,19 @@ int fi_ibv_open_ep(struct fid_domain *domain, struct fi_info *info,
 	struct fi_info *fi;
 	int ret;
 
+	size_t len = 40;
```

Comment:
Why is this 40? Add some comments here to describe why the length is 40.

Reply:
Good point. I'll create a proper define that documents the length.

Reply:
The OFI define OFI_ADDRSTRLEN is the correct length specifier and will be used.